poetry_scripts.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+
+def format_code():
+    os.system("poetry run black .")
```

Comment:
super minor: current dir is relative to the file

Reply:
Good catch. Current implementation only works from the root dir. We should look at improving the stability of this command in the future.